src/partner/partner.controller.ts

```diff
@@ -33,10 +35,13 @@ export class PartnerController {
   }
 
   @Get(':name')
-  @ApiOperation({ description: 'Retuns profile data for a partner' })
+  @ApiOperation({ description: 'Returns profile data for a partner' })
+  @UseGuards(SuperAdminAuthGuard) // Temporary super admin auth guard
```

Comment:
I assume this is a temporary guard right?

Reply:
Yes